README.md

```diff
@@ -11,47 +15,66 @@ This project was inspired by Arsunt's
 [TR2Main](https://github.com/Arsunt/TR2Main/) project and the legacy name,
 "Tomb1Main", reflects that.
 
-## Installing
-
-1. Download the latest Tomb1Main release from the GitHub releases.  
-   https://github.com/rr-/Tomb1Main/releases
+## Showcase
 
-2. Unpack the contents to your game directory.  
+<table>
+    <tr>
+        <th>
+            Mode selection
+            <img src="docs/showcase1.jpg"/>
+        </th>
+        <th>
+            Braid
+            <img src="docs/showcase2.jpg"/>
+        </th>
+    </tr>
+    <tr>
+        <th>
+            Fly cheat
+            <img src="docs/showcase3.jpg"/>
+        </th>
+        <th>
+            Door open cheat
+            <img src="docs/showcase4.jpg"/>
+        </th>
+    </tr>
+    <tr>
+        <th>
+            Enemy healthbars
+            <img src="docs/showcase5.jpg"/>
+        </th>
+    </tr>
+</table>
+
+## Installing (simplified)
+
+1. Head over to GitHub releases: https://github.com/rr-/Tomb1Main/releases
+2. Download the installer. Your browser may complain that the .exe is unsafe, but it's OK to ignore this alert.
+3. Mark the installer EXE as safe to run by right-clicking on the .exe, going to properties and clicking "Unblock".
```

Comment:
Don't need this line imo. It won't say "Unblock" for Win7 users for example, it's called something else.

Reply:
Windows 7 support ended on January 14, 2020